### PR TITLE
Final Readme.md file

### PR DESCRIPTION
**Pull request by iharshka**
Hey @arpitghura 
This is the final commit from my side, give your input also if there is a need for more changes to the readme.md file.
Regards.
Harsh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 
-Typing Test is a webapp to test and practice your typing skills in order to enhance them. 
+<img src="./assets/Homepage.png">
 
-On the App, there are automatically generating words and sentences so that there will be no repetition of sentences or words.
+Typing Test is a webapp to test and practice your typing skills in order to enhance them. In the App, words and sentences are generated randomly to ensure no repetition of a word or a sentence.
 
-The Web App can be accessible from [https://typing-test-project.vercel.app/](https://typing-test-project.vercel.app/)
+The Web App can be accessible from [https://typing-test-project.vercel.app/](https://typing-test-project.vercel.app/)</br></br>
+![Typing test](https://img.shields.io/badge/Hacktoberfest-2022-blue?style=round-square&logo=Hacktoberfest&logoColor=black&link=https://hacktoberfest.com/)
```
[the remark this answers is not in the document]
Comment (arpitghura) [reply README.md:8]: Create all the badges from [https://shields.io/](https://shields.io/) and make badge in `for-the-badge` style. 

Use these credentials:
Username: `arpitghura`
repo name: `typing-test`

**Review by arpitghura — CHANGES_REQUESTED** [review]
Make changes as requested and also create badges from the [https://shields.io/](https://shields.io/) and don't copy and paste the badges from some other repo.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+![Typing test](https://img.shields.io/badge/Contribute-red?style=round-square&logo=Github&logoColor=black&link=https://github.com/arpitghura/typing-test)
+[![Typing Test](https://img.shields.io/badge/Typing-Test-brightgreen.svg?style=flat-square)](https://makeapullrequest.com)
```
[the remark this answers is not in the document]
Comment (iharshka) [reply README.md:10]: Which one?

**Comment by iharshka** [comment]
> Make changes as requested and also create badges from the https://shields.io/ and don't copy and paste the badges from some other repo.

I have just taken a reference, thanks for sharing the link.

**Comment by iharshka** [comment]
Hey @arpitghura,
I have added the requested changes in this commit.
Changes- Moved the badges to the top, added for-the-badge style, added credentials, updated links of the badges.
Please check and do let me know.
regards

File: README.md [hunk 3]
```diff
@@ -24,18 +30,20 @@ The Web App can be accessible from [https://typing-test-project.vercel.app/](htt
...
 <img src="./assets/practice.png">
 
-Report Generates on Completion:
+Report Generated on Completion:
 
 <img src="./assets/Report.png">
 
+Thanks for coming here!
 ### Give a star if you liked this!
+#### Regards,</br>
```
Comment (arpitghura) [inline README.md:45]: Don't write these. 
Delete all after regards.

**Review by arpitghura — CHANGES_REQUESTED** [review]
Delete after regards as taught

**Comment by iharshka** [comment]
I have made the requested changes. Please check it @arpitghura

**Review by arpitghura — APPROVED** [review]
Good work

**Comment by iharshka** [comment]
Thanks arpit

**Pull request merged**